BiweeklyMeetings/2022-05-10.md

```diff
@@ -0,0 +1,494 @@
+
+**Sebastien**-:
+Yeah. But in the meantime, I think we can merge this PR because it's just a small fix.
+
+#### PR182
+[PR182 CIP-1852: fix broken CIP links](https://github.com/cardano-foundation/CIPs/pull/182)
+
+**Matthias**-:
+Yeah. It's fixing a inconsistency. So next one, PR 182. No, what is it? Yeah. 182. Which is fixing some login links yeah. Was approved a while back and we could just go for it. So the issue is that some of the CIPs reference other CIPs sometimes using relative links, which works on GitHub, but don't necessarily work on, on the website. So it's slightly better to use absolute URLs to make sure that, you know, things works on all platforms. We discussed that briefly on the hue and it's only about one of the CIPs anyway, but that's something we should be a bit careful with when merging new CIPs as well. So we've approved it. So unless someone has anything to say, also I'll just go and merge it. Okay. All good then.
+
+**M. Àngels**-:
+Now I have added to the agenda, some issues. Do you want to review this or do you want to leave them for after the discussion point, which are the CIP 50 and the one you brought at the beginning?
+
+**Matthias**-:
+Yeah. Let's first talk about CIP 50 and the audit guidelines first so that we have time. And then if time is left, we can go for issues, which are a bit more minor points.
+
+**M. Àngels**-:
+Okay.
+
+**Matthias**-:
+Do we have Kevin today or not? No...
+
+**M. Àngels**-:
+We have Michael but not Kevin.
+
+### Last Check 
+
+### Review 
+
+### Discussions 
+
+#### PR242
+ [PR242 CIP-0050? | Shelleys Voltaire decentralization update](https://github.com/cardano-foundation/CIPs/pull/242)
+
+
+**Matthias**-:
+Yeah. So maybe Michael knows, actually. So there was this discussion a few weeks ago to start a study group on CIP 50 with people from IOG and this was something Kevin picked up and was sorting. I'm not sure if it has happened yet or no. It hasn't happened yet Michael, okay. So that's something we'd need to follow up with. I know Kevin did reach out to a few people and it was acting behind the scene to get it done, but yeah, I will just reach out to him. Yes, definitely. Vasil has brought some attention already. But yeah, there are people I think we can, from which we can pick their brain to just stop thinking about that who are not directly involved in Vasil. Okay. I will just reach out to Kevin and ask him if he has any update on that regarding this study group. Maybe you can bring Michael on screen, if just for a short of date, maybe you have had also some discussions on the site already or got any feedback that you want to share?
```

Comment:
```suggestion
Yeah. So maybe Michael knows, actually. So there was this discussion a few weeks ago to start a study group on CIP 50 with people from IOG and this was something Kevin picked up and was sorting. I'm not sure if it has happened yet or no. It hasn't happened yet Michael, okay. So that's something we'd need to follow up with. I know Kevin did reach out to a few people and it was acting behind the scene to get it done, but yeah, I will just reach out to him. Yes, definitely. Vasil has brought some attention already. But yeah, there are people I think we can, from whom we can pick their brain to just start thinking about that who are not directly involved in Vasil. Okay. I will just reach out to Kevin and ask him if he has any update on that regarding this study group. Maybe you can bring Michael on screen, if just for a short time, maybe you have had also some discussions on the side already or got any feedback that you want to share?
```